homeassistant/components/forecast_solar/__init__.py

```diff
@@ -0,0 +1,77 @@
+"""The Forecast.Solar integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from forecast_solar import ForecastSolar
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_LATITUDE, CONF_LONGITUDE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import (
+    CONF_AZIMUTH,
+    CONF_DAMPING,
+    CONF_DECLINATION,
+    CONF_MODULES_POWER,
+    DOMAIN,
+)
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Forecast.Solar from a config entry."""
+    api_key = entry.options.get(CONF_API_KEY)
+    if not api_key:
+        api_key = None
```

Comment:
`get` will return `None` if it's not defined so this is not needed.

Reply:
Due to the way or option flow works, it can be an empty string as well, that is what this is catching